README.md

```diff
@@ -101,7 +101,7 @@ jobs:
           fetch-depth: 0                        # Includes all history for all branches and tags
 
       - id: get-version
-        uses: im-open/git-version-lite@v2.2.0
+        uses: im-open/git-version-lite@v2
```

Comment:
In this case I would prefer keeping the detailed action.  Not everyone will want to default to using the major version, so having this detail right on the readme is handy if you don't want to go looking for the latest tag which is extra steps.  It shouldn't be an issue either because we have automation that will automatically go in and commit the detailed version if it hasn't been added.

Reply:
I agree we should leave the specific version in the readme. in which case, the build workflow change in this PR needs to be reverted. But it would be nice to include a static note in the README (maybe a comment above the line) that says the major-version tag is available.